modular_zubbers/code/modules/vending/wardrobe.dm

```diff
@@ -8,3 +8,15 @@
 	)
 	. = ..()
 
+/obj/machinery/vending/autodrobe/Initialize(mapload)
+	products += list(
+		/obj/item/clothing/suit/costume/cardborg/fnaf/bear = 1,
+		/obj/item/clothing/head/costume/cardborg/fnaf/bear = 1,
+		/obj/item/clothing/suit/costume/cardborg/fnaf/bird = 1,
+		/obj/item/clothing/head/costume/cardborg/fnaf/bird = 1,
+		/obj/item/clothing/suit/costume/cardborg/fnaf/bunny = 1,
+		/obj/item/clothing/head/costume/cardborg/fnaf/bunny = 1,
+		/obj/item/clothing/suit/costume/cardborg/fnaf/foxy = 1,
+		/obj/item/clothing/head/costume/cardborg/fnaf/foxy = 1,
+	)
+
```

Comment:
This probably won't work with how the autodrobe uses product categories

Reply:
It didn't!!! I am unsure how else to do this. Maybe a crate. But I feel that such a lowley object is unworthy of holding the glory of fnaf.